scripts/release_ppa.sh

```diff
@@ -51,17 +58,39 @@ is_release() {
     [[ "${branch}" =~ ^v[0-9]+(\.[0-9]+)*$ ]]
 }
 
-keyid=379F4801D622CDCF
-email=builds@ethereum.org
+# source keyid and email from .release_ppa_auth
+if [[ -e .release_ppa_auth ]]
+then
+    # shellcheck source=/dev/null
+    source .release_ppa_auth
+fi
+
+[[ "$LAUNCHPAD_KEYID" != "" && "$LAUNCHPAD_EMAIL" != "" ]] ||\
+    fail "Error: Couldn't find variables \$LAUNCHPAD_KEYID or \$LAUNCHPAD_EMAIL in sourced file .release_ppa_auth (check top comment in $0 for more information)."
```

Comment:
It's a good habit to always put a space before `\`. Otherwise you might get two words fused into one without a space if you don't indent the second line :)
```suggestion
[[ "$LAUNCHPAD_KEYID" != "" && "$LAUNCHPAD_EMAIL" != "" ]] || \
    fail "Error: Couldn't find variables \$LAUNCHPAD_KEYID or \$LAUNCHPAD_EMAIL in sourced file .release_ppa_auth (check top comment in $0 for more information)."
```